support/mender-connect.service

```diff
@@ -2,7 +2,7 @@
 Description=Mender Connect service
 Wants=network-online.target
 After=systemd-resolved.service network-online.target mender-client.service
-Requires=mender-client.service
+Requires=mender-authd.service
```

Comment:
I think we should re-think this approach. It should not be mandatory to upgrade to mender 4.0.0 to keep using mender-connect. But I don't think it's possible to achieve that with `Requires`, because there is no OR.

So my proposal is: Switch to `Wants=`, which is weaker, but still strong enough for our purposes, I believe. Then you can list both services, and it won't fail to start even though one will always be missing. If both are missing, it will still start, and won't work, but I don't think this is significantly worse than not starting.

Reply:
Thank you.

I actually considered this for a long while, but eventually I thought that it should have good reasons to be already in `Requires` (for example to be stopped if `mender-client` is stopped).

But once you bring it up, and in light of being defensive from `mender-connect` point of view, it makes sense to switch to `Wants`.

Note also that I have now marked the commit as `fix` and not `feat`, so that we can (and will) cherry-pick to stable branches.